README.md

```diff
@@ -25,29 +27,52 @@ test:
     - ./cc-test-reporter after-build --exit-code $CIRCLE_EXIT_CODE
 ```
 
-*TODO*: the above is DRAFT content.
+## Code Climate: Enterprise
+
+To report coverage to your locally-hosted *Code Climate: Enterprise* instance,
+export the `CC_TEST_REPORTER_COVERAGE_ENDPOINT` variable, or pass the
+`--coverage-endpoint` option to `after-build`.
+
+```sh
+CC_TEST_REPORTER_COVERAGE_ENDPOINT=https://codeclimate.my-domain.com/test_reports
+```
 
 ## Low-level Usage
 
-See the [man-pages](man).
+The test reporter is implemented as a composition of lower-level commands, which
+may themselves be useful. See the [man-pages](man) for details of these
+commands.
 
-## Client-Side Aggregation (i.e Parallel Test Coverage)
+## Parallel Tests
 
-TODO: describe further
+Code Climate doesn't yet support receiving partial payloads from (e.g.) parallel
+test runs and summing them together server-side. However, the reporter does
+support summing them together client-side.
 
-1. After each test:
+This requires you store the partial payloads yourself after each test, then
+download them to one location before using the reporter to upload a summed
+payload.
 
-   ```
-   eval $(cc-test-reporter env)
-   cc-test-reporter format-coverage --out "coverage/codeclimate.$N.json"
-   aws s3 sync coverage/ "s3://my-bucket/coverage/$GIT_COMMIT_SHA"
-   ```
+For example:
 
-1. After all tests:
+1. After *each* test:
```

Comment:
That's a good point. I'd like to get a little further in the project (so we have a few more input formats built) before deciding exactly how to document or address this.